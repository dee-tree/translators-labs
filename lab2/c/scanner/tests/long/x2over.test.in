L#2147483648