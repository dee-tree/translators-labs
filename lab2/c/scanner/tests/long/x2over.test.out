
Fatal error: number 2147483648 is too big
